File: Chapter-4/C programming/Problem1.c
#include <stdio.h>

int main()
{

    char word;
    int a;
    scanf("%c", &word);
    a = (int)word;

    if (a >= 97 && a <= 122)
    {
        printf("%c is a Lowercase alphabet", word);
    }

    if (a >= 0 && a <= 64)
    {
        printf("%c is a Special symbol", word);
    }
    return 0;
}

File: Chapter-4/Java/Problem1.java
import java.util.Scanner;

public class Problem1 {
    public static void main(String[] args) {
        char word;
        int a;

        Scanner sc = new Scanner(System.in);
        word = sc.next().charAt(0);
        a = (int) word;
        if (a >= 97 && a <= 122) {
            System.out.println("Lowercase Alphabet");
        }

        if (a >= 0 || a <= 64) {
            System.out.println("Special Symbol");
        }

        sc.close();

    }
}